pkgs/applications/editors/nano/default.nix

```diff
@@ -32,7 +32,7 @@ in stdenv.mkDerivation rec {
     (lib.enableFeature enableTiny "tiny")
   ];
 
-  postInstall = ''
+  postInstall = if enableTiny then null else ''
```

Comment:
Does `lib.optionalString` work here?

Reply:
Obviously, but null values aren't passed to the build environment, whereas empty strings can be.